modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4174,6 +4184,34 @@ type SetupTime {
   reacquisition: TimeSpan!
 }
 
+"""
+Input parameters for setting the guide star name for an observation.
+"""
+input SetGuideTargetNameInput {
+  """
+  The program id for the observation being set.
+  """
+  programId: ProgramId!
+
+  """
+  The id of the observation to set.
+  """
+  observationId: ObservationId!
```

Comment:
Given the `observationId`, do we need the `programId` as well?  Shouldn't we allow the observation reference as an alternative as well (for example as in `CloneObservationInput`).

Reply:
Yeah. You're right. I guess I was just being lazy while I developed this. I'll go back and make those changes.